### ISSUE TITLE
Installation using brew fails with Ruby 3.0

### ISSUE THREAD

**Issue by matan7890** [apjanke/ronn-ng#78]
When installing ronn-ng using brew, I'm getting the following issue:
```
brew install ronn-ng                                                                                                                                                                                                                                                      1 ✘  12:51:00 
==> Installing ronn-ng from apjanke/ronn-ng
==> Downloading https://rubygems.org/gems/mustache-1.0.0.gem
Already downloaded: /Users/matan/Library/Caches/Homebrew/downloads/4148d48f83730eefbee6e544b01b21971db08edb1e4133b9fbcf3a166f1f00b1--mustache-1.0.0.gem
==> Downloading https://rubygems.org/gems/mini_portile2-2.4.0.gem
Already downloaded: /Users/matan/Library/Caches/Homebrew/downloads/2a6726a644744f59f1de664871c64c16eff301318a3081615f47259700559c32--mini_portile2-2.4.0.gem
==> Downloading https://rubygems.org/gems/nokogiri-1.9.0.gem
Already downloaded: /Users/matan/Library/Caches/Homebrew/downloads/9cf2d7c9477d37fe91f96a26ab814ff83e779eaa9a8ea1f672eaedd83d8c5116--nokogiri-1.9.0.gem
==> Downloading https://rubygems.org/gems/kramdown-2.1.0.gem
Already downloaded: /Users/matan/Library/Caches/Homebrew/downloads/c4a67e0c6d5121ee34876f82616518f678005b8fdc742243a09432189aee3f68--kramdown-2.1.0.gem
==> Downloading https://github.com/apjanke/ronn-ng/archive/v0.9.1.tar.gz
Already downloaded: /Users/matan/Library/Caches/Homebrew/downloads/6d31063aa889d0d81ee76a9b8b9793bd983b5366a01549d585f9bd898f470616--ronn-ng-0.9.1.tar.gz
==> Downloading https://rubygems.org/gems/mustache-1.0.0.gem
Already downloaded: /Users/matan/Library/Caches/Homebrew/downloads/4148d48f83730eefbee6e544b01b21971db08edb1e4133b9fbcf3a166f1f00b1--mustache-1.0.0.gem
==> gem install /Users/matan/Library/Caches/Homebrew/downloads/4148d48f83730eefbee6e544b01b21971db08edb1e4133b9fbcf3a166f1f00b1--mustache-1.0.0.gem --no-document --ignore-dependencies --install-dir /usr/local/Cellar/ronn-ng/0.9.1/libexec
Last 15 lines from /Users/matan/Library/Logs/Homebrew/ronn-ng/01.gem:
2021-05-12 12:52:17 +0300

gem
install
/Users/matan/Library/Caches/Homebrew/downloads/4148d48f83730eefbee6e544b01b21971db08edb1e4133b9fbcf3a166f1f00b1--mustache-1.0.0.gem
--no-document
--ignore-dependencies
--install-dir
/usr/local/Cellar/ronn-ng/0.9.1/libexec

ERROR:  Error installing /Users/matan/Library/Caches/Homebrew/downloads/4148d48f83730eefbee6e544b01b21971db08edb1e4133b9fbcf3a166f1f00b1--mustache-1.0.0.gem:
	mustache-1.0.0 requires Ruby version ~> 2.0. The current ruby version is 3.0.1.64.

If reporting this issue please do so at (not Homebrew/brew or Homebrew/core):
  https://github.com/apjanke/homebrew-ronn-ng/issues
```
It might be a problem with this mustache gem (for using ruby 2 or not specifying its dependencies correctly?), but maybe you can help me figure out how install ronn-ng?

**Comment by apjanke** [comment]
This sounds like ronn-ng needs to bump its dependency versions to handle the new Ruby 3.0 release.

I'm going to have to do a little research on how to do this in a way that supports both Ruby 2 and Ruby 3, if that's possible. It might also be that we need to stick with Ruby 2, and declare a dependency on that specific version of Ruby.

Sorry for the inconvenience here!

**Comment by matan7890** [comment]
I'm not a ruby enthusiast, but I can tell you that making it work with ruby 3 wasn't a hard problem (just bumped all the dependencies versions until it worked). 
I don't know how big the changes from ruby 2 to 3 (is it like the changes from python 2 to 3?), but I believe most of the dependencies will work on both version, am I wrong?

**Comment by apjanke** [comment]
> I don't know how big the changes from ruby 2 to 3 (is it like the changes from python 2 to 3?), but I believe most of the dependencies will work on both version, am I wrong?

I suspect you're right, but don't actually know. Guess I'll just test it and find out...

**Comment by apjanke** [comment]
Okay, with the fixes for https://github.com/apjanke/ronn-ng/issues/87, I've (I think) added support for up to Ruby 3.2, and while I can't guarantee it'll actually run right, Ronn-NG should still install and attempt to run under newer Ruby 3.x versions.

I have published an 0.10.1.pre3 ronn-ng gem to RubyGems. If no complaints in the next couple days, I'll push out a real Ronn-NG 0.10.1 release.
